cmd/dmverity-vhd/main.go

```diff
@@ -116,6 +117,10 @@ var createVHDCommand = cli.Command{
 			Name:  passwordFlag + ",p",
 			Usage: "Optional: custom registry password",
 		},
+		cli.BoolFlag{
+			Name:  hashDeviceVhdFlag,
```

Comment:
Maybe a shorthand for the flag of "hdv" to avoid typing `--hash-dev-vhd` like we have for the other flags.